### PR TITLE
build multiplat manifests with buildah

### PR DESCRIPTION
**Pull request by rbtr**
Signed-off-by: Evan Baker <rbtr@users.noreply.github.com>

**Reason for Change**:

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests

**Notes**:

### CODE REVIEW

File: .pipelines/containers/container-template.yaml [hunk 1]
```diff
@@ -13,35 +13,36 @@ steps:
...
     make ${{ parameters.name }}-image OS=${{ parameters.os }} ARCH=${{ parameters.arch }} PLATFORM=${{ parameters.os }}/${{ parameters.arch }} TAG=$PLATFORM_TAG
   name: image_build
   displayName: Image Build
   retryCountOnTaskFailure: 3
 
 - script: |
+    set -e
+    export PLATFORM_TAG=$(make container-platform-tag TAG=${{ parameters.tag }} PLATFORM=${{ parameters.os }}/${{ parameters.arch }})
+    export REF=$(IMAGE_REGISTRY)/$(make ${{ parameters.name }}-image-name):$PLATFORM_TAG
+    skopeo copy containers-storage:$REF docker-daemon:$REF
```
Comment (jaer-tsun) [inline .pipelines/containers/container-template.yaml:33]: nice!

**Review by jaer-tsun — APPROVED** [review]
lgtm

**Comment by rbtr** [comment]
patching together various runs, tests have passed

**Pull request merged**